mungegithub/bulk-lgtm/www/script.js

```diff
@@ -0,0 +1,110 @@
+"use strict";
+var app = angular.module('BulkLGTMModule', ['ngMaterial', 'angularMoment']);
+
+app.config(['$compileProvider', function ($compileProvider) {
+  $compileProvider.debugInfoEnabled(false);
+}]);
+
+app.controller('BulkLGTMCntl', ['DataService', '$interval', '$location', BulkLGTMCntl]);
+
+function BulkLGTMCntl(dataService, $interval, $location) {
+  var self = this;
+  self.prs = {};
+  self.prsCount = 0;
+  self.health = {};
+  self.metadata = {};
+  self.testResults = {};
+  self.tabLoaded = {};
+  self.functionLoading = {};
+  self.queryNum = 0;
+  self.selected = 1;
+  self.OverallHealth = "";
+  self.StatOrder = "-Count";
```

Comment:
health, testReules, tabLoaded, functionLoading, queryNum, selected, OverallHealth, StatOrder all seem to be copy/paste fluff.